frontend/templates/product/sections/ProductSection/index.tsx

```diff
@@ -106,8 +107,12 @@ export function ProductSection({
                )}
                <ProductTitle mb="2.5">{product.title}</ProductTitle>
                <ProductPrice
-                  price={selectedVariant.formattedPrice}
-                  compareAt={selectedVariant.formattedCompareAtPrice}
+                  formattedPrice={selectedVariant.formattedPrice}
+                  formattedCompareAtPrice={
+                     selectedVariant.formattedCompareAtPrice
+                  }
+                  isDiscounted={selectedVariant.isDiscounted}
+                  discountLabel={`${selectedVariant.discountPercentage}% OFF`}
```

Comment:
I'm curious why you chose to have the caller pass a formatted discount label, as opposed to having them pass the `discountPercentage` and formatting the discount label internally? It seems like a simpler API to just pass the `discountPercentage` to me.

Reply:
It was all due to [this open discussion](https://github.com/iFixit/react-commerce/issues/533)

The design proposal for prices is still not definitive. But a few scenarios make embedding logic in the component itself complicates. For instance we will probably need to show `min`/`max` prices:

<img width="490" alt="Screenshot 2022-09-01 at 00 10 20" src="https://user-images.githubusercontent.com/7805759/187795129-e1912a4b-3fa3-49d7-b375-2d8ca45c384e.png">

This leads to the fact that we will potentially need to compute the discount percentage differently, according to the context in which we are rendering the price component.
If we are showing price ranges there is also the possibility that the label needs to be a bit richer to be explicative. Imagine the case in which not all variants have the same discount

Having labels allows us to compute the values outside the component and to avoid having a lot of complex conditions inside. 

If the API will be clearer in the future and we see the possibility to embed logic inside the component I will do for sure 🚀 . But if the price in the product card ends up diverging too much we may also need to separate the two.

Reply:
Thanks for the explanation, makes complete sense!